app/src/main/java/com/beemdevelopment/aegis/ui/MainActivity.java

```diff
@@ -929,6 +932,9 @@ private void copyEntryCode(VaultEntry entry) {
         ClipboardManager clipboard = (ClipboardManager) getSystemService(Context.CLIPBOARD_SERVICE);
         ClipData clip = ClipData.newPlainText("text/plain", entry.getInfo().getOtp());
         clipboard.setPrimaryClip(clip);
+        if(_entryListView.isMinizeAfterCopyEnabled()) {
```

Comment:
I don't think the EntryListView needs to know about this. We can simply check for ``_prefs.isMinimizeOnCopyEnabled()`` here.

Also, missing a space after ``if`` here.